BLE Indoor Positioning/src/main/java/com/nexenio/bleindoorpositioning/ble/beacon/signal/SignalFilter.java

```diff
@@ -0,0 +1,62 @@
+package com.nexenio.bleindoorpositioning.ble.beacon.signal;
+
+import java.util.concurrent.TimeUnit;
+
+/**
+ * Created by leon on 09.01.18.
+ */
+
+public abstract class SignalFilter implements RssiFilter {
```

Comment:
Not too happy about `SignalFilter`, as every class that implements `RssiFilter` is a signal filter. The point of this class is to provide a sliding window, which the current name doesn't imply.